pkg/gardenlet/controller/seed/seed/reconciler_delete.go

```diff
@@ -271,6 +273,12 @@ func (r *Reconciler) runDeleteSeedFlow(
 			}).DoIf(gardenletfeatures.FeatureGate.Enabled(features.ManagedIstio)),
 			Dependencies: flow.NewTaskIDs(destroyIstio),
 		})
+		destroyFluentOperatorCRDs = g.Add(flow.Task{
+			Name: "Destroy Fluent Operator CRDs",
+			Fn: flow.TaskFn(func(ctx context.Context) error {
+				return component.OpDestroyAndWait(fluentOperatorCRDs).Destroy(ctx)
+			}),
```

Comment:
```suggestion
			Fn: component.OpDestroyAndWait(fluentOperatorCRDs).Destroy,
```

Should be enough, no?